src/style.css

```diff
@@ -0,0 +1,138 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+:root {
+  --main-colur: #060b35;
+  --grey-colour: #616070;
+  --blue-colur: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+p,
+div,
+h1,
+h2 {
+  margin: 0;
+}
+
+.laptop {
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-size: 10px ;
+  justify-content: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 0 16px 16px;
+}
+
+.card__img {
+  align-self: center;
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+}
+
+.card__name {
+  margin: 40px 0 0;
+  bottom: 165px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-colur);
+}
+
+.card__code {
+  font-style: normal;
+  font-size: 10px;
+  margin: 4px 0 0;
+  line-height: 14px;
+  width: 98px;
+  height: 14px;
+  color: var(--grey-colour);
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 16px;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-position: center;
+  flex-direction: row;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 24px;
+}
+
+.card__price-name {
+  color: var(--grey-colour);
+  font-family: inherit;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price-money {
+  color: var(--main-colur);
+  font-family: inherit;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.card__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #fff;
+  font-family: inherit;
+  font-style: normal;
+  text-decoration: none;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  background-color: var(--blue-colur);
+  border-radius: 5px;
+  width: 166px;
+  height: 40px;
+  bottom: 17px;
+  margin-top: 16px;
+}
+
+.card__button:hover {
+  color: var(--blue-colur);
+  background-color: #fff;
+  border: 1px solid var(--blue-colur);
```

Comment:
set `border: 1px solid transparent` for `.card__button` and just change the color on `hover`